industry/main.tex

```diff
@@ -8,18 +8,18 @@
 
 \subfile{components/background}
 
-\resumesection{\emoji{books} Education}
+\resumesection{\emoji{hand-with-fingers-splayed} Skills}
 
-\subfile{components/education}
+\subfile{components/skills}
 
 \resumesection{\emoji{briefcase} Experience}
 
 \subfile{components/experience/UC}
 \subfile{components/experience/IMFD}
 
-\resumesection{\emoji{hand-with-fingers-splayed} Skills}
+\resumesection{\emoji{books} Education}
 
-\subfile{components/skills}
+\subfile{components/education}
```

Comment:
I rearranged the elements to put the skill listing on top, then the experience and the education at the bottom. The logic behind this is to help the recruiter or whoever is doing the screening find your skillset easily.
People will glance at the resume so I advise putting the technical skills and background first to make a better impression. 
In my experience, screeners don't put a lot of weight on the Education background